### PR TITLE
introduce OpenAI ChatGPT connector

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

File: market/connector/openai-connector/meta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "id": "openai-connector",
+    "name": "OpenAI's ChatGPT API",
+    "description": "OpenAI’s ChatGPT is a text-based, general-purpose productivity application that perfectly fits to process automation.",
```
Comment (ivy-rew) [inline market/connector/openai-connector/meta.json:4]: as proposed by @andreasbalsiger

File: market/connector/openai-connector/logo.png
Comment (ivy-rew) [inline market/connector/openai-connector/logo.png:1]: https://commons.wikimedia.org/wiki/File:ChatGPT_logo.svg

File: market/connector/openai-connector/meta.json (hunk 1 above)
Comment (andreasbalsiger) [reply market/connector/openai-connector/meta.json:4]: great

**Review by andreasbalsiger — APPROVED** [review]
lets go

**Pull request merged**